tremor-script/src/grammar.lalrpop

```diff
@@ -1421,6 +1588,15 @@ extern {
         "use" => Token::Use,
         "as" => Token::As,
 
+        "pipeline" => Token::Pipeline,
+        "connector" => Token::Connector,
+        "flow" => Token::Flow,
+        "links" => Token::Links,
+        "connect" => Token::Connect,
+        "to" => Token::To,
+        "deploy" => Token::Deploy,
+        "config" => Token::Config,
```

Comment:
Maker to remember, not a review comment: this is a great list of new keywords that introduce breaking changes.

Reply:
I added an entry to `CHANGELOG.md` so we don't need to remember